### ISSUE TITLE
Quarantine MapAction_JsonBodyOrService_HandlesBothJsonAndService

### ISSUE THREAD

**Issue by adityamandaleeka** [dotnet/aspnetcore#47274]
## Failing Test(s)

`Microsoft.AspNetCore.Http.Generators.Tests.RequestDelegateCreationTests.MapAction_JsonBodyOrService_HandlesBothJsonAndService`

## Error Message

```text
Line 92 does not match.
Expected Line:
[(@"TestMapActions.cs", 24)] = (
Actual Line:
[(@"TestMapActions.cs", 25)] = (
Expected: True
Actual:   False
```

## Stacktrace

<details>

```text

   at Microsoft.AspNetCore.Http.Generators.Tests.RequestDelegateCreationTestBase.VerifyAgainstBaselineUsingFile(Compilation compilation, String callerName) in /_/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTestBase.cs:line 321
   at Microsoft.AspNetCore.Http.Generators.Tests.RequestDelegateCreationTests.MapAction_JsonBodyOrService_HandlesBothJsonAndService() in /_/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.JsonBodyOrService.cs:line 76
--- End of stack trace from previous location ---
```
</details>

## Logs

<details>

```text

```
</details>

## Build

Looks like this is failing consistently in every aspnetcore-ci run.

https://dev.azure.com/dnceng-public/public/_build/results?buildId=208119&view=ms.vss-test-web.build-test-results-tab&runId=3857770&resultId=106995&paneView=debug

**Comment by adityamandaleeka** [comment]
@captainsafia Do we need to regenerate the baseline?

**Comment by adityamandaleeka** [comment]
Currently failing on:
https://github.com/dotnet/aspnetcore/pull/47247
https://github.com/dotnet/aspnetcore/pull/47271
https://github.com/dotnet/aspnetcore/pull/47272

**Comment by mitchdenny** [comment]
Should be fixed when #47272 merges. I'll keep an eye on it and merge it as soon as I get a green pass. I think the array PR and and the update to the way we do service resolution in RDG didn't play nicely together in main.

**Comment by mitchdenny** [comment]
Should be resolved now.
